src/diffusers/schedulers/scheduling_ddim_flax.py

```diff
@@ -104,6 +104,7 @@ class FlaxDDIMScheduler(SchedulerMixin, ConfigMixin):
             `set_alpha_to_one=False`, to make the last step use step 0 for the previous alpha product, as done in
             stable diffusion.
     """
+    has_state = True
```

Comment:
Think this should be a  read-only `property` instead of class variable so it can't be modified by users.